extension/js/common/browser/ui.ts

```diff
@@ -210,6 +212,8 @@ export class Ui {
     },
   };
 
+  public static testCompatibilityLink = '<a href="/chrome/settings/modules/compatibility.htm" target="_blank">Test your private key compatibility</a>';
```

Comment:
Let's rename to `Test your OpenPGP key compatibility` (in case we use it for testing public keys too, which we may)

Reply:
Done :+1: